test/specs/lib/getKeyDownHandlers-test.ts

```diff
@@ -33,7 +33,7 @@ const eventArg: React.KeyboardEvent = {
   target: null,
   timeStamp: null,
   type: null,
-}
+} as any
```

Comment:
@sophieH29 to take a look and how we can consolidate `KeyboardEvent` and `React. KeyboardEvent` interfaces